homeassistant/components/ohme/binary_sensor.py

```diff
@@ -0,0 +1,90 @@
+"""Platform for binary_sensor."""
+
+from __future__ import annotations
+
+import logging
+
+from homeassistant.components.binary_sensor import (
+    BinarySensorDeviceClass,
+    BinarySensorEntity,
+    BinarySensorEntityDescription,
+)
+from homeassistant.config_entries import ConfigEntry
+from homeassistant.const import EntityCategory
+from homeassistant.core import HomeAssistant
+from homeassistant.helpers.entity_platform import AddEntitiesCallback
+
+from .entity import OhmeEntity
+
+_LOGGER = logging.getLogger(__name__)
+
+BINARY_SENSOR_DESCRIPTIONS = [
+    BinarySensorEntityDescription(
+        key="car_connected",
+        icon="mdi:ev-plug-type2",
+        device_class=BinarySensorDeviceClass.PLUG,
+    ),
+    BinarySensorEntityDescription(
+        key="car_charging",
+        icon="mdi:battery-charging-100",
+        device_class=BinarySensorDeviceClass.BATTERY_CHARGING,
+    ),
+    BinarySensorEntityDescription(
+        key="pending_approval",
+        icon="mdi:alert-decagram",
+    ),
+    BinarySensorEntityDescription(
+        key="charger_online",
+        icon="mdi:web",
+        device_class=BinarySensorDeviceClass.CONNECTIVITY,
+        entity_category=EntityCategory.DIAGNOSTIC,
+    ),
```

Comment:
I would rather bake this into the availbility of your base entity, as if your charger is offline probably all entities get unavailable anyway